### ISSUE TITLE
Linux VM installation fails using install.sh script

### ISSUE THREAD

**Issue by judin** [chatwoot/chatwoot#5281]
Following the instructions https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm on a barebones fresh install of Ubuntu 20.04 on a Linode 8GB.

➥ 1/9 Installing dependencies. This takes a while.
➥ 2/9 Installing databases.
➥ 3/9 Installing webserver.
➥ 4/9 Setting up Ruby
➥ 5/9 Setting up the database.
➥ 6/9 Installing Chatwoot. This takes a long while.
➥ 7/9 Running database migrations.

Some error has occured. Check '/var/log/chatwoot-setup.log' for details.

Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js??ref--3-1!node_modules/vue-loader/lib/loaders/stylePostLoader.js!node_modules/postcss-loader/src/index.js??ref--3-2!node_modules/sass-loader/dist/cjs.js??ref--3-3!node_modules/vue-loader/lib/index.js??vue-loader-options!app/javascript/widget/components/template/Article.vue?vue&type=style&index=0&id=feefd62e&lang=scss&scoped=true&:
    Entrypoint mini-css-extract-plugin = *
       2 modules
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js??ref--3-1!node_modules/vue-loader/lib/loaders/stylePostLoader.js!node_modules/postcss-loader/src/index.js??ref--3-2!node_modules/sass-loader/dist/cjs.js??ref--3-3!node_modules/vue-loader/lib/index.js??vue-loader-options!app/javascript/widget/components/template/EmailInput.vue?vue&type=style&index=0&id=a4d16ac0&lang=scss&scoped=true&:
    Entrypoint mini-css-extract-plugin = *
       2 modules

rails aborted!
ActiveRecord::ConnectionNotEstablished: FATAL:  database "chatwoot" does not exist
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:21:in `block (3 levels) in <main>'
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:18:in `each'
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:18:in `block (2 levels) in <main>'

Caused by:
PG::ConnectionBad: FATAL:  database "chatwoot" does not exist
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:21:in `block (3 levels) in <main>'
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:18:in `each'
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:18:in `block (2 levels) in <main>'
Tasks: TOP => db:chatwoot_prepare
(See full trace by running task with --trace)

**Comment by vishnu-narayanan** [comment]
Confirmed Linux installation is breaking for `v2.8.0` install.

ref: https://github.com/chatwoot/chatwoot/runs/7869274506?check_suite_focus=true#step:6:31

**Comment by vishnu-narayanan** [comment]
@judin Could you please try this?

```
sudo -i -u chatwoot
vi .env.example
# comment or remove the POSTGRES_DATABASE variable
exit
# retry the installer
./install.sh --install
```

**Comment by judin** [comment]
Thanks @vishnu-narayanan. env.example is an empty file. Still the same problems.

I used Ubuntu 20.04 as it was what you used in your example. I can try with Ubuntu 22.04?

**Comment by vishnu-narayanan** [comment]
@judin Its `.env.example` under `chatwoot` directory. Could you `cd` after switching to `chatwoot` user?

```
sudo -i -u chatwoot
cd chatwoot
vi .env.example
```

**Comment by vishnu-narayanan** [comment]
>I used Ubuntu 20.04 as it was what you used in your example. I can try with Ubuntu 22.04?
This installer is tested and maintained on `20.04` as of now. If `22.04` is a requirement, you could try out the docker deployment method.

**Comment by judin** [comment]
That fixed in on 20.04 - thank you!

**Comment by vishnu-narayanan** [comment]
Fixed in Chatwoot Release `2.8.1`
